states/statefile/testdata/roundtrip/v3-simple.in.tfstate

```diff
@@ -87,4 +88,4 @@
             "depends_on": []
         }
     ]
-}
+}
```

Comment:
Nope, this isn't making a difference -  just an auto-formatting thing 🤷‍♀

Reply:
Indeed, the round-trip test this is consumed by decodes both the before and after and compares them as a decoded data structure rather than comparing the JSON byte-for-byte, so the exact formatting of these files is not significant for the sake of that test:

https://github.com/hashicorp/terraform/blob/02850111b98c8b2b844b6695a100cecdc0d9afd4/states/statefile/roundtrip_test.go#L59-L73

I think for some reason our text editors just have some disagreement about whether text files ought to end with a newline or not, which is a little strange since we're both using the same editor (assuming I wrote these files originally, which I think I did?) but something we can dig into some more another day.